src/v2i-hub/CDASimAdapter/src/include/CDASimConnection.hpp

```diff
@@ -112,21 +115,37 @@ namespace CDASimAdapter {
              * @return Returns true if CARMA-Simulation connection is active and false if the connection is inactive.
              */
             bool is_connected() const;
+
         private:
+            /**
+             * @brief helper method to generate JSON used for handshake connection with CARMA-Simulation instance. 
+             * @param local_ip address of infrastructure software (V2X-Hub).
+             * @param time_sync_port port assigned to listening for time sychronization messages from CARMA-Simulation.
+             * @param v2x_port port assigned to listening for v2x messages from CARMA-Simulation.
+             * @param location simulated location of infrastructure hardware.
+             * @return true if handshake successful and false if handshake unsuccessful.
+             */
+            std::string get_handshake_json(const uint infrastructure_id, const std::string &local_ip,  const uint time_sync_port, 
```

Comment:
This method should be constant since it does not change the underlying object.

Reply:
changed